src/bridge/cockpitdbusjson.c

```diff
@@ -1294,6 +1305,11 @@ handle_dbus_call (CockpitDBusJson *self,
       cockpit_channel_fail (COCKPIT_CHANNEL (self), "protocol-error",
                             "the \"flags\" field is invalid in dbus call");
     }
+  else if (!cockpit_json_get_int(object, "timeout", G_MAXINT64, &call->timeout))
```

Comment:
The maximum / default value should be G_MAXINT. Because this is used as a gint when passing off to DBus.